pkg/deploy/configuration.go

```diff
@@ -245,7 +246,17 @@ func (d *DataMongoConfigure) Configure(client Client, deployment *dc.DeploymentC
 		}
 		return url
 	}
-	dataDc, err := client.FindDeploymentConfigsByLabel(namespace, map[string]string{"rhmap/name": "data"})
+	// look for the Job if it already exists no need to run it again
+	existingJob, err := client.FindJobByName(namespace, deployment.Name+"-dataconfig-job")
+	if err != nil {
+		d.statusUpdate(deployment.Name, "error finding existing Job "+err.Error(), "error")
```

Comment:
Should this be an error, that seems a bit alarming. Could it not say "could not find existing Job, will launch new one" or something like that?